cmake/Dependencies.cmake

```diff
@@ -1266,7 +1266,7 @@ if (NOT INTERN_BUILD_MOBILE)
     SET(AT_CUDA_ENABLED 1)
   endif()
 
-  IF (NOT AT_CUDA_ENABLED OR NOT CUDNN_FOUND)
+  IF (NOT AT_CUDA_ENABLED OR NOT CUDNN_FOUND OR NOT $ENV{USE_CUDNN})
```

Comment:
Could you test the variable `USE_CUDNN` instead of the env variable? The env variable is passed down to CMake for the first time then it persistently sits in CMakeCache.txt

Reply:
In addition, the status message won't look correct.

Reply:
sure, I will